### ISSUE TITLE
Can we have code greyed out when it commented by other commands

### ISSUE THREAD

**Issue by oziphantom** [sjPlot/Relaunch64#28]
For example 
.comment
code
code
code
.endc

all the code would be the grey colour like a normal comment?

Also something like .if 0 ?

TASS Mode.

**Comment by sjPlot** [comment]
So `.comment` and `.endc` indicate a block comment and are correct assembler syntax or commands?

**Comment by oziphantom** [comment]
Yes, in TASS64. Documented here http://tass64.sourceforge.net/#misc

**Comment by sjPlot** [comment]
Does @soci64 probably know how to deal with this issue?

**Comment by oziphantom** [comment]
I would think so. 

The .comment one should be easy, if you have an internal state in the colour highlighting system, you can just add a force comment flag, which is incremented on .comment and then decremented on .endc 

.if 0 is a little more complex and would need to do some form of expression parsing
